indico/modules/vc/templates/manage_event_select.html

```diff
@@ -1,21 +1,30 @@
-{% extends 'layout/base.html' %}
-
-{% block title %}{% trans %}Video services{% endtrans %}{% endblock %}
-
-{%- block content %}
-    {%- if plugins %}
-        <p>{% trans %}The following video services are available. Click on one to create a new room:{% endtrans %}</p>
-        <div class="i-badges-list-left i-badges-list-hover">
-            {%- for plugin in plugins %}
-                <a class="js-plugin-details i-badge i-badge-gray" href="{{ url_for(vc_room_action, event, service=plugin.service_name) }}">
-                    <div class="i-badge-content">
-                        <span class="i-badge-img">
-                            <img src="{{ plugin.logo_url }}">
-                        </span>
-                        <span class="i-badge-title">{{ plugin.title }}</span>
-                    </div>
-                </a>
-            {%- endfor %}
-        </div>
-    {%- endif %}
-{%- endblock %}
+{%- if plugins %}
+    <p>{%  set action=_("create") if vc_room_action==".manage_vc_rooms_create" else _("attach") %}
+       {% trans %} The following video services are available. Click on one to {{ action }} a new room:{% endtrans %}
+    </p>
+    <div class="i-badges-list-left i-badges-list-hover">
+        {%- for plugin in plugins %}
+            <a href="#" class="js-vc-plugin i-badge i-badge-gray"
+               data-href="{{ url_for(vc_room_action, event, service=plugin.service_name) }}"
+               {%  set action=_("Create") if vc_room_action==".manage_vc_rooms_create" else _("Attach") %}
```

Comment:
too much space before `set`, also move this above the `<a href ...` line. having this in the middle of a tag definition is ugly